extensions/logging/src/interceptors/logging.interceptor.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/extension-logging
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  asGlobalInterceptor,
+  bind,
+  BindingScope,
+  config,
+  ContextTags,
+  inject,
+  Interceptor,
+  InvocationContext,
+  Provider,
+  ValueOrPromise,
+} from '@loopback/context';
+import {RequestContext, RestBindings} from '@loopback/rest';
+import * as morgan from 'morgan';
+import {format} from 'util';
+import {Logger} from 'winston';
+import {LoggingBindings} from '../keys';
+
+/**
+ * A local interceptor that provides logging for method invocations.
+ */
+@bind({
+  tags: {[ContextTags.KEY]: LoggingBindings.WINSTON_INTERCEPTOR},
+  scope: BindingScope.SINGLETON,
+})
+export class LoggingInterceptor implements Provider<Interceptor> {
```

Comment:
Please find a name that better communicates what this interceptor logs. For example: `LogInvocationArgumentsInterceptor`.

Reply:
Renamed to WinstonInvocationLogger